### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by GregShiner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ Welcome to the Better Bar Crawl App! This application allows users to discover a
 ### Clone the Repository
 
 ```bash
-git clone https://github.com/yourusername/BetterBarCrawlApp.git
-cd BetterBarCrawlApp
+git clone https://github.com/pwjensen/BetterBarCrawlApp.git
+cd bar-crawl-app
```
Comment (pwjensen) [inline README.md:36]: Incorrect I think

**Review by pwjensen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pwjensen) [reply README.md:36]: Line 36

Comment (GregShiner) [reply README.md:36]: image
image

Comment (GregShiner) [reply README.md:36]: image

Comment (GregShiner) [reply README.md:36]: unless other git implementations do some wonky stuff

Comment (pwjensen) [reply README.md:36]: so line 36 is incorrect, if it's added

**Review by TamaraEstrada — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GregShiner) [reply README.md:36]: me when im stupid

**Review by pwjensen — APPROVED** [review]
YAY

**Pull request merged**